Chapter3/Chapter3.Task3/Program.cs

```diff
@@ -8,12 +8,16 @@ static void Main(string[] args)
         {
             Console.WriteLine("Hello World!");
             int num1 = 10, num2 = 2;
```

Comment:
please read task description carefully. You should read out 2 numbers from console(NOT HARDCODE them).
Read 2 strings and convert them to numbers